tomcat/src/main/java/org/apache/coyote/http11/reqeust/QueryParameters.java

```diff
@@ -0,0 +1,37 @@
+package org.apache.coyote.http11.reqeust;
```

Comment:
```suggestion
package org.apache.coyote.http11.request;
```

패키지 명에 오타가 있는 것 같아요!